src/components/View.tsx

```diff
@@ -0,0 +1,41 @@
+import '../styles/View.scss';
+
+const viewGridGreenIcon = '/images/grid-green.svg';
+const viewGridGreyIcon = '/images/grid-grey.svg';
+const viewListGreenIcon = '/images/list-green.svg';
+const viewListGreyIcon = '/images/list-grey.svg';
+
+type ViewProps = {
+    view: string,
+    changeView: Function,
+}
+
+const View = ({ view, changeView }: ViewProps) => {
```

Comment:
View - is too generic name. It better to specify what this view is going to be user for